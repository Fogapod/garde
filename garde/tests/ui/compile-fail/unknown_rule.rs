#![allow(dead_code)]

#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(ascii, unknown_rule)]
    field: &'a str,
}

fn main() {}
