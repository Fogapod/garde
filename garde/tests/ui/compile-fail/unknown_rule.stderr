error: unrecognized validation rule
 --> tests/ui/compile-fail/unknown_rule.rs
  |
  |     #[garde(ascii, unknown_rule)]
  |                    ^^^^^^^^^^^^
